src/actions/CreateAccountActions.ts

```diff
@@ -110,8 +110,9 @@ export function createUser(data: CreateUserData) {
     setTimeout(async () => {
       try {
         const abcAccount = await context.createAccount(
-          data.username,
-          data.password,
+          // HACK: Temporary workaround pending optional username support
+          data.username ?? `edge_light_account_${Date.now()}`,
```

Comment:
Is there a core PR for this? Perhaps this PR should depend on that

Reply:
Not at the time of writing but there is now